public/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Proof-of-authority (PoA)
+description: An explanation of the proof-of-authority consensus protocol and its role in blockchain ecosystem.
+lang: en
+---
+
+Proof-of-Authority is a reputation-based consensus algorithm that is a modified version of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). It is mostly used by private chains, testnets, and local development networks. PoA is a reputation-based consensus algorithm that requires trusting a set of authorized signers to produce blocks, instead of a stake-based mechanism in PoS.
```

Comment:
Similar to "proof-of-stake" which we abbreviate "PoS'... and capitalizing the first word only as it starts the sentence. Adding the PoS abbreviation and bolding since it's the first time we're introducing the term/abbreviation on the page.

```suggestion
**Proof-of-authority (PoA)** is a reputation-based consensus algorithm that is a modified version of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). It is mostly used by private chains, testnets, and local development networks. PoA is a reputation-based consensus algorithm that requires trusting a set of authorized signers to produce blocks, instead of a stake-based mechanism in PoS.
```